src/test/java/io/reactivex/internal/operators/observable/ObservableSingleTest.java

```diff
@@ -282,6 +282,7 @@ public void testSingleWithEmpty() {
 
         InOrder inOrder = inOrder(observer);
         inOrder.verify(observer).onComplete();
+        inOrder.verify(observer, never()).onError(any(Throwable.class));
```

Comment:
This is actually covered by `verifyNoMoreInteractions` but being explicit here doesn't hurt.